src/components/authorization/policies/by-role/marketing.policy.ts

```diff
@@ -18,5 +33,6 @@ import { member, Policy, Role, sensMediumOrLower, sensOnlyLow } from '../util';
       p.marketingLocation.edit,
     ])
     .children((c) => c.posts.edit),
+  r.StepProgress.whenAll(member, variant('official')).read,
```

Comment:
Marketing as to be a member of the project to see progress, that doesn't seem right

Reply:
That's how the spreadsheet reads; I'll double-check.

Reply:
It's supposed to be they can see any low sensitivity projects.